en/identity-server/next/docs/references/service-extensions/pre-flow-extensions/pre-issue-access-token-action/api/pre-issue-access-token-action-v1.yaml

```diff
@@ -412,6 +442,19 @@ components:
             - type: array
               items:
                 type: string
+    RefreshTokenClaims:
+      type: object
+      properties:
+        name:
+          type: string
+        value:
+          oneOf:
+            - type: string
+            - type: integer
+            - type: boolean
+            - type: array
+              items:
+                type: string
```

Comment:
Do we need to allow all these ? It's just expiry we allow to change right

Reply:
Fixed fa91ed432a84540f87b805e17e9e977b56cea654